src/locales/en/translation.json

```diff
@@ -1 +1,223 @@
-../../../public/locales/en/translation.json
```

Comment:
ah, this was the symlink that was making problems. I'm not sure what a goof solution for this is, but could you revert this change? You can replace the symlink again, just not commit it